components/bl-social-login-button-component/src/index.js

```diff
@@ -0,0 +1,22 @@
+import { Button } from './button';
+
+const { cn } = BackendlessUI.CSSUtils
+
+export default function SocialLoginButtonComponent({ component, eventHandlers }) {
+  const { display, classList, disable, showButtonIcon, googleplus, facebook, twitter, linkedin, github } = component;
+  const { onClick } = eventHandlers;
+
+  if (!display) {
+    return null;
+  }
+
+  return (
+    <div className={ cn('bl-customComponent-socialLoginButton', classList, { disable }) }>
+      { googleplus && <Button socialNet={ "googleplus" } showButtonIcon={ showButtonIcon } onClick={ onClick } /> }
```

Comment:
We must pass which type of login used in onClick, also that mean need to add context block 'Used Login' or 'Login Type' at this handler.